templates/payments/refund-request.html

```diff
@@ -12,18 +12,31 @@ <h2>Refund your payment</h2>
 <p>If you have any questions,
    please <a href="mailto:{{ config['TICKETS_EMAIL'][1] }}?subject=Refund Query (WEB-{{payment.id}})">drop us an email</a>.</p>
 
+<p>We cannot refund tickets you have transferred to others or checked in. If you have transferred tickets please request them back
+   and we can then refund them.</p>
+
+{% if REFUND_STATE=="illness" %}
+  <p>We are currently only issuing refunds in case of illness.</p>
+{% endif %}
+
 <div class="well">
 <form method="post" action="{{ url_for('.payment_refund_request', payment_id=payment.id, currency=currency) }}">
 {{ form.hidden_tag() }}
-  <fieldset>
-    <legend>Donation</legend>
-    <p>If you want, you can donate some of your payment to help us recoup our losses for this year
-        and fund our next event. This is completely optional &mdash; we have enough money in the bank
-        to survive until 2022.</p>
-    {{ render_field(form.donation_amount, min=0, max=payment.amount, value=0,
-          donation_range_payment=payment, donation_range_currency=CURRENCY_SYMBOLS[currency]) 
-    }}
-  </fieldset>
+
+
+  <p>Please select any and all purchases you would like to receive a refund for.</p>
```

Comment:
Not sure if this is necessary or just nitpicking, but I'd suggest:

```suggestion
  <p>Please select any and all purchases you would like to request a refund for.</p>
```

Also not 100% sure whether it should be plural (`a refund for` / `refunds for`).  I'll try to consult a language reference after some other code review.

Reply:
I suppose that in the case of multiple selected items, a single refund is issued, so in that sense singular (`a refund`) is correct.